app/pb/account.proto

```diff
@@ -33,6 +34,19 @@ service Account {
   rpc MarkContributorFormFilled(MarkContributorFormFilledReq) returns (google.protobuf.Empty) {
     // Marks the contributor form as filled
   }
+
+  rpc ChangePhone(ChangePhoneReq) returns (google.protobuf.Empty) {
+    // Set/Change/Remove phone number
+  }
+
+  rpc SendVerificationSMS(SendVerificationSMSReq) returns (SendVerificationSMSRes) {
+    // Send phone number verification sms. The code in the SMS should be send back in VerifyPhone()
+  }
```

Comment:
Why have these as different RPCs? I think it overly complicates the state machine, easier to just have one RPC that sets the pending number and sends a verification code, then another one to verify the sent code itself.

Reply:
Since you only have one try per year, I thought it would be neat to see your own phone number shown by the app before pushing the verify button. But you are right it complicates the state machine a bit.

Reply:
> Since you only have one try per year, I thought it would be neat to see your own phone number shown by the app before pushing the verify button.

We should do that in the frontend!

Reply:
Yay!